CIP-XXXX/cardano-babbage.json

```diff
@@ -0,0 +1,1826 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "cardano-babbage.json",
```

Comment:
It would be nice to have a property for version/hardfork of the schema as a property in the object